src/components/Header.tsx

```diff
@@ -31,10 +56,19 @@ export default function Header() {
           <div className="relative w-max space-x-14 flex top-2">
             {headerSubSections.map((section, index) => (
               <button
-                onMouseEnter={handleMouseEnter}
-                onMouseLeave={handleMouseLeave}
+                onMouseEnter={() => {
+                  handleMouseEnter();
+                  changeHeaderContent(section.text);
+                }}
+                onMouseLeave={() => {
+                  handleMouseLeave();
+                  changeHeaderContent(section.text);
+                }}
                 className="text-sm py-2.5 text-center inline-flex items-center hover:text-orange-600"
-                onClick={handleDropDown}
+                onClick={() => {
+                  changeHeaderContent(section.text);
+                  handleDropDown();
```

Comment:
This seems like an odd function, what does handleDropDown really do if it doesn't handle the change in content? Feels like the change in header should be moved over to handleDropDown

Reply:
addressed in different branch.